pom.xml

```diff
@@ -922,6 +928,34 @@
             </plugins>
          </build>
       </profile>
+      <profile>
+          <id>java9on</id>
+          <activation>
+              <jdk>[9,)</jdk>
+          </activation>
+          <properties>
+              <maven.compiler.source>8</maven.compiler.source>
+              <maven.compiler.target>11</maven.compiler.target>
```

Comment:
Profile for 9+ but target for 11 seems odd?

I get why its targeting 11 just now since we unfortunately cant use the -release config at the moment to compile for 8, and so would instead need to edit the sources to have things work on 8 when compiled with 9+. 

I'd probably just use enforcer to prevent use of 9 and 10 at this point given I dont think it ever built on them and they are already long EOL.